client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/transaction/SyncModeTransactionTable.java

```diff
@@ -81,24 +84,26 @@ private SynchronizationAdapter createSynchronizationAdapter(Transaction transact
    private static class SynchronizationAdapter implements Synchronization {
 
       private final Map<String, TransactionContext<?, ?>> registeredCaches = new ConcurrentSkipListMap<>();
-      private final Collection<TransactionContext<?, ?>> preparedCaches = new LinkedList<>();
       private final Transaction transaction;
       private final Consumer<Transaction> cleanupTask;
       private final RemoteXid xid;
+      private final long timeout;
 
-      private SynchronizationAdapter(Transaction transaction, Consumer<Transaction> cleanupTask, RemoteXid xid) {
+      private SynchronizationAdapter(Transaction transaction, Consumer<Transaction> cleanupTask, RemoteXid xid,
+            long timeout) {
          this.transaction = transaction;
          this.cleanupTask = cleanupTask;
          this.xid = xid;
+         this.timeout = timeout;
       }
 
       @Override
       public String toString() {
          return "SynchronizationAdapter{" +
-               "registeredCaches=" + registeredCaches.keySet() +
-               ", transaction=" + transaction +
-               ", xid=" + xid +
-               '}';
+                "registeredCaches=" + registeredCaches.keySet() +
+                ", transaction=" + transaction +
+                ", xid=" + xid +
+                '}';
```

Comment:
I see you've started aligning wrapped lines to the expression start in some cases, but not yet everywhere, e.g. with parameters... don't worry, you'll get there ;)

Reply:
every time I update intellij it decides to change my format config. I gave up...